web/blog/vast-v2.3/index.md

```diff
@@ -0,0 +1,55 @@
+---
+draft: true
+title: VAST v2.3
+description: Automatic Rebuilds
+authors: dominiklohmann
+date: 2022-08-05
+tags: [release, rebuild, performance]
+---
+
+We released [VAST v2.3][github-vast-release]! 
+
+FIXME: Write a short summary.
+
+[github-vast-release]: https://github.com/tenzir/vast/releases/tag/v2.3.0
+
+<!--truncate-->
+
+## Automatic Rebuilds
+
+VAST server processes now continually rebuild outdated and merge undersized
+partitions in the background. To control this behavior, set the new
+`vast.automatic-rebuild` option.
+
+```yaml
+vast:
+  # Control automatic rebuilding of partitions in the background for
+  # optimization purposes. The given number controls how many rebuilds to run 
+  # concurrently, and thus directly controls the performance vs. memory and CPU
+  # usage trade-off. Set to 0 to disable. Defaults to 1.
+  automatic-rebuild: 1
+```
+
+This has two major advantages:
+
+1. For VAST users, optimally sized partitions in the most recent will be the
+   norm. This improves query performance throughout the board.
```

Comment:
```suggestion
   norm. This improves query performance throughout the board.
```

Please add sentence with an explanation, along the lines of: "...because of the reduced overall number of partitions"